management/src/main/java/org/ehcache/management/registry/DefaultManagementRegistryConfiguration.java

```diff
@@ -29,22 +30,29 @@
 /**
  * @author Ludovic Orban
  */
-public class DefaultManagementRegistryConfiguration implements ManagementRegistryConfiguration {
+public class DefaultManagementRegistryConfiguration implements ManagementRegistryServiceConfiguration {
 
   private static final AtomicLong COUNTER = new AtomicLong();
 
   private final Map<Class<? extends ManagementProvider>, StatisticsProviderConfiguration<?>> configurationMap = new HashMap<Class<? extends ManagementProvider>, StatisticsProviderConfiguration<?>>();
-  private String cacheManagerAlias;
+  private Context context = Context.empty();
   private String statisticsExecutorAlias;
+  private String collectorExecutorAlias;
 
   public DefaultManagementRegistryConfiguration() {
-    // defaults
-    this.cacheManagerAlias = "cache-manager-" + COUNTER.getAndIncrement();
+    setCacheManagerAlias("cache-manager-" + COUNTER.getAndIncrement());
     addConfiguration(new EhcacheStatisticsProviderConfiguration(1, TimeUnit.MINUTES, 100, 1, TimeUnit.SECONDS, 30, TimeUnit.SECONDS));
   }
 
   public DefaultManagementRegistryConfiguration setCacheManagerAlias(String alias) {
-    this.cacheManagerAlias = alias;
+    return setContext(Context.create("cacheManagerName", alias));
+  }
+
+  public DefaultManagementRegistryConfiguration setContext(Context context) {
+    if (!this.context.contains("cacheManagerName") && !context.contains("cacheManagerName")) {
```

Comment:
this.context.contains("cacheManagerName") || context.contains("cacheManagerName")